docs/updating-the-about-us-page.md

```diff
@@ -0,0 +1,47 @@
+# Updating the About Us Page
+
+## Adding a new staff member
+
+The list of staff members can be found at [/about](https://www.thunderbird.net/en-US/about/) on the website.
+
+To add a new staff member, you only need to:
+
+- add the headshot image files
+- update the markup in the About Us template
+
+### Headshots
+
+When exporting a staff member's headshot from a photo editing tool, make sure to create two versions: one at 1x resolution and another at 2x resolution.
+
+Follow these naming convention:
+
+- 1x: `roc.png`
+- 2x: `roc-high-res.png`
+
+Note: The high resolution naming convention deviates from the industry standard "@2x".
+
+Save both to `media/img/thunderbird/staff/`.
+
```

Comment:
We have a tool for automagically cropping and compressing staff list images
https://github.com/thunderbird/thunderbird-website/blob/master/tools/crop_staff_list.py

To use it simply cd into tools, and run `python ./crop_staff_list.py` it will automagically crop and compress every image in `media/img/thunderbird/staff/uncropped` (which I think is gitignored.) It has some extra options for specifying the origin point of the crop by setting `[Left|Center|Right,Top|Center|Bottom]` (ex/ `[Left,Bottom]` in the filename.)

Pretty handy! Due to how I previously got the files, the anything before a - will be ignored.

Reply:
Very nice!